dashboarddoc.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: dashboard
+permalink: /dashboarddoc/
+---
+
+User Dashboard Interaction
+==========================
+{: .anchor}
+
+This document illustrates how a Solar Forecast Arbiter user may use the
+Dashboard to complete the following actions:
+
+-   [Creating a new Site](#creating-a-site).
+
+-   [Creating a new Observation or Forecast](#creating-an-observation-or-forecast).
+
+-   [Uploading Data to an Observation or Forecast](#uploading-data).
+
+When a user is filling out a form, a tooltip describing the information
+a user should enter is shown when the field’s name or options are not
+descriptive enough alone. An example of these tooltips is shown below.
+
+<img class="my-3" src="/images/TooltipDetail.png" class="figure"/>
+
+Creating a Site
+---------------
+{: .anchor}
+
+1. Navigate to sites listing page using ‘Sites’ link in the left sidebar. At the top of the Site listing click ‘Create new Site’.
+<img class="my-3" src="/images/SitesListing.png"/>
+
+2. Enter the metadata for your Site. Selecting a site type of 'Power Plant' will prompt you for additional fields as shown below. Sites that are power plants will require  After submission, you will be redirected to a Site page which displays the new Site’s metadata. The Site page will also display links to pages listing its associated Observations and Forecasts. See [Creating an Observation or Forecast](#creating-an-observation-or-forecast).
```

Comment:
"Sites that are power plants will require  After submission, you will be redirected to a Site page which displays the new Site’s metadata. The Site page will also display links to pages listing its associated Observations and Forecasts. See [Creating an Observation or Forecast](#creating-an-observation-or-forecast)."

becomes

"After submission, you will be redirected to a Site page which displays the new Site’s metadata and allows you to create associated Observations and Forecasts. See [Creating an Observation or Forecast](#creating-an-observation-or-forecast)."